sdk/cognitivelanguage/Azure.AI.Language.Conversations/api/Azure.AI.Language.Conversations.netstandard2.0.cs

```diff
@@ -7,13 +7,11 @@ public partial class AnalysisParameters
     }
     public partial class AnalyzeConversationOptions
     {
-        public AnalyzeConversationOptions(string projectName, string deploymentName, string query) { }
-        public string DeploymentName { get { throw null; } }
+        public AnalyzeConversationOptions(string query) { }
```

Comment:
As mentioned above, move this out to its own param *only* and hide it from this class.